Mobile Buy SDK/Mobile Buy SDK Tests/BUYClientTest.m

```diff
@@ -327,14 +327,17 @@ - (void)testProductsInCollectionWithSortOrderTitleDescending
 
 - (void)testCustomerCreationURL
 {
-	BUYAccountCredentialItem *firstName = [BUYAccountCredentialItem itemWithKey:@"first_name" value:@"michael"];
-	BUYAccountCredentialItem *lastName = [BUYAccountCredentialItem itemWithKey:@"last_name" value:@"scott"];
-	BUYAccountCredentialItem *email = [BUYAccountCredentialItem itemWithKey:@"email" value:@"fake@example.com"];
-	BUYAccountCredentialItem *password = [BUYAccountCredentialItem itemWithKey:@"password" value:@"password"];
-	BUYAccountCredentialItem *passwordConfirmation = [BUYAccountCredentialItem itemWithKey:@"password_confirmation" value:@"password"];
-	BUYAccountCredentials *credentials = [BUYAccountCredentials credentialsWithItems:@[firstName, lastName, email, password, passwordConfirmation]];
-	
-	NSURLSessionDataTask *task = [self.client createCustomerWithCredentials:credentials callback:nil];
+	BUYAccountCredentials *credentials = [BUYAccountCredentials credentialsWithItems:@[
+																					   [BUYAccountCredentialItem itemWithFirstName:@"michael"],
+																					   [BUYAccountCredentialItem itemWithLastName:@"scott"],
+																					   [BUYAccountCredentialItem itemWithEmail:@"fake@example.com"],
+																					   [BUYAccountCredentialItem itemWithPassword:@"password"],
+																					   [BUYAccountCredentialItem itemWithPasswordConfirmation:@"password"],
+																					   ]];
+	
+	NSURLSessionDataTask *task = [self.client createCustomerWithCredentials:credentials callback:^(BUYCustomer * _Nullable customer, NSString * _Nullable token, NSError * _Nullable error) {
```

Comment:
Not sure why it shows up with this excessive indentation. It looks more civilized in Xcode. The nullability annotation is inserted by code completion so it wasn't intentional, but I agree its way more cluttered with it. Will fix both.